tgui/packages/tgui/interfaces/IntegratedCircuit/index.js

```diff
@@ -256,14 +260,15 @@ export class IntegratedCircuit extends Component {
                     position="absolute"
                     top={0}
                     color="transparent"
-                    onClick={() => act('save_circuit')}
+                    onClick={() => act("save_circuit")}
```

Comment:
Wow, changing single quotes to double quotes... Our "prettier" formatter is set up to prefer single quotes, so I'd rather follow that.

Reply:
Part of the same recently-merged PR by Mothblocks, https://github.com/tgstation/tgstation/compare/master...Gurkenglas:n-to-n-wires doesn't show this.

Reply:
I think you're looking at a merge commit. Github is bad at showing those. If you want to look at the way I resolved the conflicts in that merge, try locally checking out the merge commit and doing `git show -cc`.